tests_new/integration_tests/aws_clients/iam.py

```diff
@@ -21,13 +21,12 @@ def get_role(self, role_name):
             log.info(f'Error occurred: {e}')
```

Comment:
nit: is `def create_role_if_not_exists(...)` method used or only `def get_consumption_role(...)` to get or create the iam role

If not used lets remove